spec/javascripts/support/jasmine.yml

```diff
@@ -7,7 +7,9 @@ src_files:
 css_files:
  - "support.{css,css.sass}"
 
-spec_dir: ../spec/javascripts
+spec_dir: 
+  - "../spec/javascripts"
+  - "../*/spec/javascripts"
```

Comment:
@searls sure thing! I couldn't think of a good way to point directly to the myengine example here while also testing globbing and having this make sense (e.g. both `../myengine/*/javascripts` and `../myengine/spec/*` don't look right). I ended up adding the engine under `engines/myengine` and changing the `spec_dir` here to `../engines/*/spec/javascripts` which I think is more clear. Let me know if that works. Thanks for your feedback.